sonar-php-plugin/src/main/java/org/sonar/plugins/php/phpunit/xml/TestSuite.java

```diff
@@ -100,8 +106,8 @@ public TestSuite(final String name, final String file, final String tests, final
     this.tests = tests;
     this.assertions = assertions;
     this.time = time;
-    this.testSuites = testSuites;
-    this.testCases = testCases;
+    this.testSuites = CollectionUtils.isEmpty(testSuites) ? Collections.<TestSuite>emptyList() : testSuites;
+    this.testCases = CollectionUtils.isEmpty(testCases) ? Collections.<TestCase>emptyList() : testCases;
```

Comment:
@kinow Hi! 
Why this change? I can't reproduce any problem when `testsuite` has 0 `testsuite` child elements. Anyway it will be valuable to test this change.